tests/integration/targets/nomad/aliases

```diff
@@ -0,0 +1,5 @@
+shippable/posix/group2
```

Comment:
Please rename the directory to `nomad_job` and add
```suggestion
shippable/posix/group2
nomad_job_info
```
so that the tests are run for changes to any of the two modules.